racetrack_commons/racetrack_commons/plugin/call.py

```diff
@@ -0,0 +1,30 @@
+from inspect import getfullargspec
+from typing import Callable, Dict, Any
+
+from racetrack_client.log.logs import get_logger
+
+logger = get_logger(__name__)
+
+
+def safe_call(function: Callable, **kwargs):
```

Comment:
Ahh, you even made it generic. I thought about just checking for that single argument, but this is much better. Safety and future-proofness. I give you :bacon: